assets/js/googlesitekit/modules/components/DefaultModuleSetup.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * DefaultModuleSetup component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../../components/button';
+
+export default function DefaultModuleSetup( { finishSetup, module = {} } ) {
```

Comment:
I think we should use minimal props here like we usually do and only pass the `moduleSlug` which is sufficient to identify the module. Then this component can look up the module object to get `module.name` from the datastore.